src/assets/styles/theme.scss

```diff
@@ -1,6 +1,17 @@
 // FONTS
 $helvetica: Helvetica, sans-serif;
 
+//FONT SIZES
```

Comment:
dej tu tylko proszę spacje by to nei było sklejone i mozna mergować
sry ale mnie to razi xD